crates/figma_import/src/bin/fetch.rs

```diff
@@ -86,14 +90,18 @@ fn fetch_impl(args: Args) -> Result<(), ConvertError> {
         id: doc.get_document_id(),
     };
     println!("Fetched document");
+    println!("  DC Version: {}", SerializedDesignDocHeader::current().version);
     println!("  Doc ID: {}", doc.get_document_id());
     println!("  Version: {}", doc.get_version());
     println!("  Name: {}", doc.get_name());
     println!("  Last Modified: {}", doc.last_modified().clone());
     // We don't bother with serialization headers or image sessions with
```

Comment:
This comment is no longer accurate, correct?

Reply:
Updated comment.